### DISCUSSION TITLE
Airtable to MysQL using NocoDB - issue [Jun 18]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2485] [Discord Threads]
> 🙋‍♂️ **ilyak**

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### please paste the project info  
#  
> 💁 **o1lab**  
#### right topmenu  
#  
> 💁 **o1lab**  
#### as of now we are allowing dump of airtable to nocodb  
#  
> 💁 **o1lab**  
#### and there is not periodic sync  
#  
> 💁 **o1lab**  
#### and you will have to do that manually  
#  
> 💁 **o1lab**  
#### please DM me  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/987305141340344340)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
